contracts/interfaces/modules/royalty/policies/IIpRoyaltyVault.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+/// @title Ip royalty vault interface
+interface IIpRoyaltyVault {
+    /// @notice Event emitted when a claim is made
+    /// @param claimerIpId The claimer ipId address
+    event Claimed(address claimerIpId);
+
+    /// @notice Event emitted when a snapshot is taken
+    /// @param snapshotId The snapshot id
+    /// @param snapshotTimestamp The timestamp of the snapshot
+    /// @param unclaimedTokens The amount of unclaimed tokens at the snapshot
+    event SnapshotCompleted(uint256 snapshotId, uint256 snapshotTimestamp, uint32 unclaimedTokens);
+
+    /// @notice initializer for this implementation contract
+    /// @param name The name of the royalty token
+    /// @param symbol The symbol of the royalty token
+    /// @param supply The total supply of the royalty token
+    /// @param unclaimedTokens The amount of unclaimed royalty tokens reserved for ancestors
+    /// @param ipIdAddress The ip id the royalty vault belongs to
+    function initialize(
+        string memory name,
+        string memory symbol,
+        uint32 supply,
+        uint32 unclaimedTokens,
+        address ipIdAddress
+    ) external;
+
+    /// @notice Adds a new revenue token to the vault
+    /// @param token The address of the revenue token
+    /// @dev Only callable by the royalty policy LAP
+    function addIpRoyaltyVaultTokens(address token) external;
```

Comment:
do you need to emit an event for this function?

Reply:
There may not be a need given we have the function `getVaultTokens` which returns that information